manifests/mirror.pp

```diff
@@ -86,6 +98,18 @@
     $components_arg = " ${components}"
   }
 
+  if empty($filter) {
+    $filter_arg = ''
+  } else{
+    $filter_arg = "-filter=\"${filter}\""
```

Comment:
Thanks Dean,

Sorry I didn't come back to you - I see one of your colleagues has opened another PR with this change - https://github.com/gds-operations/puppet-aptly/pull/64 so I'll close mine.

Regards,
Trevor